packages/viewer/src/components/viewer-transform-widget/viewer-transform-widget.tsx

```diff
@@ -34,6 +34,18 @@ export class ViewerTransformWidget {
   @Event({ bubbles: true })
   public positionChanged!: EventEmitter<Vector3.Vector3 | undefined>;
 
+  /**
+   * An event that is emitted when the delta changed
+   */
+  @Event({ bubbles: true })
+  public deltaChanged!: EventEmitter<Matrix4.Matrix4 | undefined>;
```

Comment:
How would you feel about calling this `interactionEnded` and providing a transform? Just to fit with the naming convention of the other event 🙂